# Robot Information
int32[] RobotNumber
int32[] Fitness
int32[] AvFitness
int32 Generation
# Chromosome Information
float32[] SensorActivation
float32[] LinearVelocity
float32[] AngularVelocity
float32[] RotationTime
float32[] SensorAngle
bool[] Mutation
# {[5],[5],...,[5]} (each robot uses 5 indexes(one for gene))
